### DISCUSSION TITLE
Socks5 forward remote to local

### DISCUSSION THREAD

**Discussion by GektorUA** [erebe/wstunnel#236] [Q&A]
I have VPS server, and i want make SOCKS5 chain from local PC to remote VPS. So:
on my client side i have SOCKS5 port, and i have started instance of wstunnel on VPS, how i can start wstunnel on client side so that on VPS side was opened my locally (clients) SOCKS5 port?

**Comment by erebe** [comment]
Hello,

with `-R socks5://0.0.0.0:1080` for example. The tunnel will listen on your VPS on port 1080
